examples/MEE-CBC/RCPA_CMA.ec

```diff
@@ -571,7 +571,7 @@ theory EtM.
           wp; sp. exists* ek{1}, p0{1}; elim* => _k _p.
           call (_: ={glob E, k, p} /\ k{1} = _k /\ p{1} = _p ==> ={glob E, res} /\ dec _k res{1} = Some _p).
             by conseq (_: ={glob E, k, p} ==> ={glob E, res}) (enc_sem _k _p); proc true.
-          by skip; smt.
+          skip => />; progress; smt(@FSet).
```

Comment:
`progress` is deprecated. I am surprised that you need it after the use of `/>`

Reply:
Yes, I was also surprised to see that neither `skip => />; smt.` nor `skip => />; smt(@FSet).` succeeds, given that the `/>` cuts down the size of the goal by about half.

Reply:
This is likely one of those instances where simply introducing the memory or memories at the head of the conclusion speeds things up.

`progress` has the side effect of introducing them.

Reply:
In this case it also splits a large goal into several smaller ones, each presumably getting the full smt time allowance.

Reply:
by skip=> />; smt(in_fsetU in_fset1).